### ISSUE TITLE
Auto Sniper/Flipper - Auto Buy/Sell feature 

### ISSUE THREAD

**Issue by taiwor88** [NeurProjects/neur-app#6]
**Is your feature request related to a problem? Please describe.**

Need a dynamic auto flipper , auto sniper and auto sell agent just like Griffain and what TrenchAI is about to release for Neura as well 

**Describe the solution you'd like**

Need a dynamic auto flipper , auto sniper and auto sell agent just like Griffain and what TrenchAI is about to release for Neura as well 

**Describe alternatives you've considered**

**Additional context**

**Comment by taiwor88** [comment]
NA

**Comment by slimeonmyhead** [comment]
Depends on #20 , this should be pretty straightforward once there are basic automation flows

**Comment by akiraloli** [comment]
Yeah, I'm bumping this. It'd be cool to snipe CA's that get posted on Twitter automatically. For example, we choose what Twitter we want it to scrape the CA from, and when they post a CA, it snipes automatically, and it can auto-sell.
